capabilities/connect.capabilities.yml

```diff
@@ -139,6 +139,170 @@ properties:
+        type: string
+        enum:
+          - ICCP
+          - Backup-Link
+          - None
+      failover-time:
+        description: Specifies the time required for traffic to fail over to the secondary switch when there is a failure in the primary switch.
+        type: integer
+        unit: "milliseconds"
+      vlan-synchronization:
+        description: Indicates whether VLAN synchronization across MC-LAG peers is supported and the number of VLANs that can be synchronized.
+        type: boolean
+      max-mac-entries-per-mclag:
+        description: Maximum number of MAC address entries that can be synchronized across MC-LAG peers.
+        type: integer
+  lldp-capabilities:
+    type: object
+    description:
+      Description of LLDP capabilities across different switch models/vendors.
+    properties:
+      supported-tlvs:
+        type: array
+        items:
+          type: string
+          enum:
+          - management-ip-address
+          - port-description
+          - system-capabilities
+          - system-description
+          - system-name
+          - proto-ident
+          - proto-vid
+          - pvid
+          - vlan-name
+          - link-agg
+          - mac-phy
+          - max-frame
+          - poe
+          - med-location
+          - med-inventory
+          - med-capabilities
+          - med-network-policy
+      lldp-med-support:
+        type: boolean
+        description: Specifies whether LLDP-MED support is available on the switch.
+        examples:
+        - true
+        - false
+      max-neighbors:
+        type: integer
+        description: Maximum number of LLDP neighbors a switch can discover and maintain.
+        examples:
+        - 64
+        - 128
+        - 256
+      refresh-interval:
+        type: object
+        description: Time interval (in seconds) for sending LLDP advertisements.
+        properties:
+          min:
+            type: integer
+            examples:
+            - 2
+            - 5
+          max:
+            type: integer
+            examples:
+            - 60
+            - 120
+      holdtime-multiplier-range:
+        type: object
+        description: Range of holdtime multiplier values for LLDP TTL calculation.
+        properties:
+          min:
+            type: integer
+            examples:
+            - 2
+            - 4
+          max:
+            type: integer
+            examples:
+            - 10
+            - 20
+      reinit-delay:
+        type: object
+        description: Time delay (in seconds) before reinitializing LLDP after a port/link down event.
+        properties:
+          min:
+            type: integer
+            examples:
+            - 1
+            - 2
+          max:
+            type: integer
+            examples:
+            - 5
+            - 10
+      notification-interval:
```

Comment:
again time value if we are going to use this

Reply:
This was removed last week when i raised second review. I'll push changes now today, for you to recheck final changes.